lib/workflow/export_test.go

```diff
@@ -0,0 +1,7 @@
+package workflow
```

Comment:
```suggestion
package workflow_test
```
Shouldn't this helper be part of the test package?